backends/platform/sdl/opendingux/opendingux.mk

```diff
@@ -1,7 +1,7 @@
 OD_EXE_STRIPPED := scummvm_stripped$(EXEEXT)
 bundle = od-opk
-ifeq ($(OPENDINGUX_TARGET), rg99)
-OPKNAME = rg99_rs90
+ifeq ($(OPENDINGUX_TARGET), rs90)
+OPKNAME = rs90_rg99
 else
```

Comment:
I think we had a discussion to change it to rs90 but I don’t really care

Reply:
Yes, please avoid this special case for the buildbot.
We will add a note on packages list to inform that it covers rg99 as well.

Reply:
Ok

Reply:
Do we still need this if? Looks like second case covers everything

Reply:
Yes that's what I meant by removing the special case :D

Reply:
Yes, indeed, done.